uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/mocks/apis/leave-request-api-mock.js

```diff
@@ -13,7 +13,15 @@ define([
       return {
         all: function (filters, pagination, sort, params) {
           return $q(function (resolve, reject) {
-            resolve(mockData.all().values);
+            var list = mockData.all().values;
+
+            resolve({
+              list: list,
+              total: list.length,
+              allIds: list.map(function (leaveRequest) {
+                return leaveRequest.id;
+              }).join(',')
```

Comment:
It would be a syntax error, you are inside an object definition, it would be equivalent to doing
```js
{
  foo: 'foo',
  bar: 'bar';
}
```